src/mscorlib/src/System/Collections/Hashtable.cs

```diff
@@ -66,6 +66,7 @@ namespace System.Collections
     // 
     [DebuggerTypeProxy(typeof(System.Collections.Hashtable.HashtableDebugView))]
     [DebuggerDisplay("Count = {Count}")]
+    [Serializable]
```

Comment:
TimeZone is currently not serializable because of it: https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/CurrentSystemTimeZone.Cache.cs#L33